.github/workflows/ci.yml

```diff
@@ -1,152 +1,84 @@
 name: ci
 on:
+  pull_request:
   push:
     branches:
     - master
-  pull_request:
-    types:
-    - opened
-    - reopened
-    - synchronize
 jobs:
-  linux-tests:
-    name: linux tests
-    runs-on: ubuntu-18.04
+  tests:
+    name: tests
+    strategy:
+      matrix:
+        # os: [ubuntu-18.04, macos-10.14, windows-2016]
+        os: [ubuntu-18.04, macos-10.14]
+        # channel: [stable, beta, nightly]
+        channel: [nightly]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
     env:
-      DISPLAY: ":99"
```

Comment:
Isn't this env variable needed anymore? This is useful for Linux runs.

Reply:
That env variable is set in the action to allow atom to run on Linux. We don't need to set it in two places.

Reply:
I see. Yes, that works!

Reply:
@UziTech Please resolve this.